public_html/wp-content/plugins/wordcamp-forms-to-drafts/wordcamp-forms-to-drafts.php

```diff
@@ -269,6 +269,8 @@ protected function simulate_post_type( $post_type ) {
 	 *   taxonomy and the selected term is applied to the drafted post. Maybe need to send PR to add filter to
 	 *   insert custom fields programmatically.
 	 * - Sideload the logo from submitted URL and set it as the featured image.
+	 * - Add a Company Description field to the form stub, or populate content from something else?
```

Comment:
I'm assuming this is for the `call for sponsors` stub, but it's probably worth making that explicit.

Reply:
Good call.